packages/esm-active-visits-app/src/home-page-tiles/total-visits-metric-tile/total-visits.resources.tsx

```diff
@@ -0,0 +1,19 @@
+import { openmrsFetch, restBaseUrl, type Visit } from '@openmrs/esm-framework';
```

Comment:
This also looks like a duplication of contents in [total-visits-tile.resources.tsx](https://github.com/openmrs/openmrs-esm-patient-management/pull/1075/files#diff-f5d913bf450c34dffd2c630ed3572e4f95024b5205cb53d2b04f8df9d4fda169)